packages/website-frontend/index.html

```diff
@@ -2,8 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Try out the Stryker dashboard, home for your test quality" />
+    <link rel="icon" type="image/svg+xml" href="/assets/favicon.ico" />
```

Comment:
Is this an svg or an ico? The type says svg, but the extension says ico. You can set an svg as the "preferred" favicon, with ico as a fallback:

```html
<link rel="icon" href="/favicon.ico" sizes="48x48">
<link rel="icon" href="/favicon.svg" sizes="any" type="image/svg+xml">
```

Reply:
It’s an .ico file, I copied this from the old implementation 😅